### PR TITLE
allowing to override module_name

### PR DESCRIPTION
**Pull request by K-Jo**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -103,9 +103,10 @@ def test_bundle_from_cmd():
     min_redis_version = "4.6"
     min_redis_pack_version = "5.0"
     display_name = "test_module"
-    os_list = ["ubuntu14.04"]
+    module_name = "module_test"
+    os_list = ["ubuntu14.04","rhel7"]
 
-    argv = [MODULE_FILE_PATH, '-a', author, '-e', email, '-D', description, '-d', display_name, '-ol', os_list,
+    argv = [MODULE_FILE_PATH, '-a', author, '-e', email, '-D', description, '-d', display_name, '-n', module_name,
```
Comment (swilly22) [inline test.py]: Why did you omitted `os_list` ?

Comment (K-Jo) [reply test.py]: Nice catch.  I added it back in and made sure the argument is different from the default

File: test.py [hunk 2]
```diff
@@ -103,19 +103,23 @@ def test_bundle_from_cmd():
...
+            '-o', BUNDLE_ZIP_FILE, '-cc', CONFIG_COMMAND]
 
     runner = CliRunner()
     result = runner.invoke(ramp.pack, argv)
 
     assert result.exit_code == 0
     metadata, _ = unpacker.unpack(BUNDLE_ZIP_FILE)
 
-    assert metadata["module_name"] == "graph"
+    assert metadata["module_name"] == module_name
```
Comment (gkorland) [inline test.py]: @K-Jo how did it pass the test before this change?

**Comment by gkorland** [comment]
@swilly22 can you please release it and publish it on pypi?

**Pull request merged**